UnitTests/Views/ShortcutTests.cs

```diff
@@ -314,4 +314,260 @@ public void Focus_CanFocus_CommandView_Add_Tracks ()
         Assert.False (shortcut.CanFocus);
         Assert.True (shortcut.CommandView.CanFocus);
     }
+
+    [Theory]
+    //  0123456789
+    // " C  0  A "
+    [InlineData (-1, 0)]
+    [InlineData (0, 1)]
+    [InlineData (1, 1)]
+    [InlineData (2, 1)]
+    [InlineData (3, 1)]
+    [InlineData (4, 1)]
+    [InlineData (5, 1)]
+    [InlineData (6, 1)]
+    [InlineData (7, 1)]
+    [InlineData (8, 1)]
+    [InlineData (9, 0)]
+    [AutoInitShutdown]
+    public void MouseClick_Fires_Accept (int x, int expectedAccept)
+    {
+        Toplevel current = new Toplevel ();
+        var shortcut = new Shortcut
+        {
+            Key = Key.A,
+            Text = "0",
+            Title = "C"
+        };
+        current.Add (shortcut);
+
+        Application.Begin (current);
+
+        int accepted = 0;
+        shortcut.Accept += (s, e) => accepted++;
+
+        Application.OnMouseEvent (new MouseEvent ()
+        {
+            Position = new Point (x, 0),
+            Flags = MouseFlags.Button1Clicked,
+        });
+
+        Assert.Equal (expectedAccept, accepted);
+
+        current.Dispose ();
+    }
+
+    [Theory]
+    //  0123456789
+    // " C  0  A "
+    [InlineData (-1, 0, 0)]
+    [InlineData (0, 1, 1)]
+    [InlineData (1, 0, 1)] // BUGBUG: This should be 1,1,1. We need to fix the logic in the Shortcut class.
```

Comment:
As mentioned here (re-posting here in case the commit comment is not easily visible):

https://github.com/tig/Terminal.Gui/commit/5cf556c376ea4f41399fb323a265f05e776e8641#r143586671

When you want to skip a test temporarily, feed a string to the Skip property, in the attribute constructor, as shown below and in the linked commit comment.

It's something attribute constructors can do that normal ones can't, and is basically like using an initializer.

For this one, this code, instead of the line with the comment and altered data to make it pass, will instead skip and print a message, without failing and without having to feed it a fake value that may or may not continue to pass if other things change:

```csharp
[InlineData (1, 1, 1, Skip = "// BUGBUG: We need to fix the logic in the Shortcut class.")]
```

Reply:
And it doesn't lie about the test results, either. 😅

Reply:
I didn't know that was there til just when I looked at this.

But it occurred to me that NUnit has that capability, and it isn't exactly an uncommon thing to need to do, so surely xUnit must have it?

And sure enough, it does. :)

Reply:
Actually, you know what?

I'll make a tag for the todo list specifically for skipped tests that looks for something like `\[\((?:.*Skip *\= *\")(.*)\" *\)\]` (or whatever I have to tweak that to, to make it just match the string in the actual regex).

That'll give them their own category so they don't add noise to the others.

Reply:
Booooooo. The ToDo list feature does not support matching in generic code elements, so even though this pattern quite nicely matches the string after a `Skip = "`, it doesn't work to tag it because it isn't a comment, identifier, or string.

That said, I can just make it look for a specific string in them, like "Skipping".

Oh well.

I'll just let strings match then. Same tags available.